spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -2294,6 +2308,12 @@ def get_current_time(self):
     def get_generated_output(self, name_of_variable):
         if name_of_variable in self._last_run_outputs:
             return self._last_run_outputs[name_of_variable]
+        if name_of_variable in self._mapping_outputs:
+            return self._mapping_outputs[name_of_variable]
+        if name_of_variable in self._load_outputs:
+            return self._load_outputs[name_of_variable]
+        if name_of_variable in self._machine_outputs:
+            return self._machine_outputs[name_of_variable]
```

Comment:
I don't see what uses this now.  Is there a reason for adding these cases?

Reply:
I don't see any resolution to this yet...

Reply:
I would say that if the change isn't needed for the rest of the changes in here, it probably should be removed.  If we really need it, it should probably go in when we do...

Reply:
so you want to keep a method buggy?

Reply:
"Buggy"... Well that depends on if you think it is a bug!  I can see what you mean but I think the definition of this is the output from the last call to run (even if the method name isn't explicit).  The other contexts are really for passing from state-to-state than for outside access...

Reply:
if ya view is that the name needs changing over actually making it support access to the data whenever. 

then ill remove and add a if for has ran (or whatever the flag is)